<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cImportExcelDialog</class>
 <widget class="QDialog" name="cImportExcelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_lpFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_lpFileNameBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_lpOpen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>open</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sheet:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_lpSheet"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="m_lpFieldList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="m_lpTest">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="m_lpDataList">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cImportExcelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cImportExcelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
